examples/wordle/README.md

```diff
@@ -39,7 +39,7 @@
 │  └── styles
 │      └── style.css <-- CSS styles for the app.
 │  └── App.tsx <-- The main app. Contains structure for the sidebar when launched.
-│      main.tsx <-- Initializes app.
+│      main.tsx <-- Initializes the git app.
```

Comment:
is this correct?

Reply:
@marcospinello / @mirabalani was it added by one of you?